userguide/directoryservice.rst

```diff
@@ -140,9 +140,8 @@ these settings by checking
    | Enable Monitoring        | checkbox      |             | Restart Active Directory automatically if the service is disconnected.                                              |
    |                          |               |             |                                                                                                                     |
    +--------------------------+---------------+-------------+---------------------------------------------------------------------------------------------------------------------+
-   | Encryption Mode          | drop-down     | ✓           | Choices are *Off*, *SSL*, or *TLS*. Refer to `SSL vs. TLS                                                           |
-   |                          | menu          |             | <https://www.globalsign.com/en/blog/ssl-vs-tls-difference/>`__ for more information on SSL and TLS.                 |
-   |                          |               |             |                                                                                                                     |
+   | Encryption Mode          | drop-down     | ✓           | Choices are *Off*, *SSL*, or *TLS*.                                                                                 |
+   |                          |               |             | Choosing SSL uses LDAPS protocol (port 636). Choosing TLS uses LDAP protocol (port 389).                            |
```

Comment:
We don't need to say "choosing" here, it's better with current tense. And single stars around field contents:
```suggestion
   |                          |               |             | *SSL* selects LDAPS protocol (port 636). *TLS* selects LDAP protocol (port 389).                            |
```